.github/workflows/tests.yml

```diff
@@ -15,7 +15,6 @@ on:
       - main
 
 env:
-  SCCACHE_GHA_ENABLED: "true"
```

Comment:
Hmm I think jobs that run on GH-hosted runners still need it

Reply:
Indeed. I am too trigger-happy.

Reply:
It should be fine now.